### DISCUSSION TITLE
会和"Fatkun 智能下载器"这个扩展程序冲突

### DISCUSSION THREAD

**Discussion by Tsubamei** [ShunCai/QZoneExport#61] [General]
当和这个扩展程序同时启用时,会导致下载的图片全部被重命名,下载失败.

**Comment by ShunCai** [comment]
> 当和这个扩展程序同时启用时,会导致下载的图片全部被重命名,下载失败.

不仅仅是这个下载器，只要拥有下载功能的扩展同时启用两个，都会冲突，这是浏览器的设计。
